config/config_default-test.php

```diff
@@ -48,7 +48,7 @@
         'settings' => [
             'charset' => 'utf8',
```

Comment:
Oh, I see you support PostgreSQL out of the box as well. You need to make sure that people will remove that `queries` when they want to use PostgreSQL. For PostgreSQL you need to tell the connection in a other way to use utf8, which you can again by setting `'charset' => 'utf8'`. So I'd let that charset as is. It doesn't hurt for MySQL when it's there but is necessary for PostgreSQL. Either way, you need to clarify that topic in the docs for people switching to pgsql.